userspace/libsinsp/event.cpp

```diff
@@ -1402,6 +1404,55 @@ Json::Value sinsp_evt::get_param_as_json(uint32_t id, OUT const char** resolved_
 	return ret;
 }
 
+std::string sinsp_evt::get_cwd(uint32_t id, sinsp_threadinfo *tinfo)
```

Comment:
Okay, maybe `get_cwd` isn't the right name, we're not getting the work directory here. No idea though what would work better, `get_base_dir`?